ch14/bias_modeling.md

```diff
@@ -1 +1 @@
-!INCLUDE "../notebooks-html/bias_modeling.html"
+!INCLUDE "../notebooks-html\bias_modeling.html"
```

Comment:
Hmm. It looks like the reverse backslashes `\` should be regular slashes `/`. This looks like a issue with running the conversion scripts on Windows. For now, can you replace the `\` with `/` for this `.md` file?